String/FirstUniqueCharacter.js

```diff
@@ -12,8 +12,12 @@ const firstUniqChar = (str) => {
   if (typeof str !== 'string') {
     throw new TypeError('Argument should be string')
   }
-  for (let i = 0; i < str.length; ++i) {
-    if (str.indexOf(str[i]) === str.lastIndexOf(str[i])) return i
+  const count = new Map()
+  for (const char of str) {
+    count[char] ? count[char]++ : count[char] = 1
```

Comment:
Could you please use `if`-`else` statements for readability instead of a ternary (since you don't use the value of the expression)?